web/onos-gui/src/app/onos-topo/entity-detail/entity-detail.component.html

```diff
@@ -63,19 +67,26 @@ <h2 class="changes-title">Relationships</h2>
                     <tr>
                         <td class="label" width="110">Source</td>
                         <td class="label" width="110">Target</td>
-                        <td class="label" width="110">Relationship</td>
                     </tr>
                 </tbody>
                 <tbody *ngFor="let relation of topoDeviceService.relationshipsList | keyvalue">
                     <tr *ngIf="relation.value.getRelation().getSrcEntityId() === detailsData?.getId()">
                         <td class="relation-value" width="110"> </td>
-                        <td class="relation-value" width="110"> {{ relation.value.getRelation().getTgtEntityId() }} </td>
-                        <td class="relation-value" width="110"> {{ relation.key }} </td>
+                        <td class="relation-value" width="110">
+                            <a class='entity-link'
+                                (click)="changeEntity(relation.value.getRelation().getSrcEntityId())"> >
+                                {{ relation.value.getRelation().getTgtEntityId() }}
+                            </a>
+                        </td>
                     </tr>
                     <tr *ngIf="relation.value.getRelation().getTgtEntityId() === detailsData?.getId()">
-                        <td class="relation-value" width="110"> {{ relation.value.getRelation().getSrcEntityId() }} </td>
+                        <td class="relation-value" width="110">
+                            <a class='entity-link'
+                                (click)="changeEntity(relation.value.getRelation().getSrcEntityId())">{{ relation.value.getRelation().getSrcEntityId() }}
+                                >
+                            </a>
+                        </td>
                         <td class="relation-value" width="110"> </td>
```

Comment:
I think this line is not doing anything

Reply:
Just kept it in because there should be two values per row in the table,
but I can remove this.